src/app/modules/shared_models/section_model.nim

```diff
@@ -111,6 +112,7 @@ QtObject:
       ModelRole.BannedMembersModel.int:"bannedMembers",
       ModelRole.Encrypted.int:"encrypted",
       ModelRole.CommunityTokensModel.int:"communityTokens",
+      ModelRole.PendingMemberRequestsModel.int:"pendingMemberRequests",
```

Comment:
These ones are needed because we had a breaking change here that prevented us from accepting membershipt requests  https://github.com/status-im/status-desktop/commit/8d955265dccaf3feec69ee675a3c3c48a05967d2#diff-4ebd6bf9f1a32ab3949e01caa71d5821f9847fe7f0a34ef3f8a96aec08c6bb10L234